_posts/dusan_andric/2016-10-28-azure-openstack.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title:  "Another cloud in the sky: Azure in a Unix shop"
+breadcrumb: true
+author: dusan_andric
+date: 2016-10-28
+categories: dusan_andric
+tags:
+    - Azure
+    - Openstack
+    - Cloud
+    - Microsoft
+teaser:
+    info: A guide for Unix shops on how to setup the Azure Storage Emulator running on your team's Openstack cloud. ...
+    image: dusan_andric/azure/azure-logo.jpg
+---
+
+## Introduction
+Microsoft Azure continues to gain popularity and as a result many teams are now working to leverage Microsoft's cloud offering for services and infrastructure. A particular point of interest is Azure Storage. 
+One question that gets asked is how do I evaluate and develop against it? An equally important question that gets asked (or should get asked) is how to I test my integrations and services? 
+And finally, will I be billed for the traffic/storage/compute while developing or running integration tests?
+
+Here Microsoft has you covered. They provide an Azure Storage Emulator that can be run locally so you can simplify your development and testing. 
+
+There is a catch however, the emulator only runs on Windows. That brings us to this guide, which should show a way forward.
+
+## The How-To
+
+### Openstack & Windows
+
+Those of you running an Openstack cloud are probably wondering how this is going to work. Thankfully a company 
+by the name of Cloudbase Solutions has solved this problem. They specialize in bringing the worlds of Openstack and Windows together. 
+Even better they provide a ready to use Windows Server 2012 Evaluation guest image. 
+
+Go ahead and get the image from here: https://cloudbase.it/windows-cloud-images/
```

Comment:
Looks like Kramdown doesn't auto convert things that look like links to links

Might as well do this then？
```
[get the image from here](https://cloudbase.it/windows-cloud-images/)
```